### DISCUSSION TITLE
Github Education Redirect to Price Page

### DISCUSSION THREAD

**Discussion by iCSawyer** [community/community#139791] [GitHub Education]
### Hi everyone,

After my application, I receive a email that tells me my application is failed and I need to reverify again. But I cannot access the price page, and the education page is redirected to the price page. Can I get the chance to apply again? It bothers me a lot!

**Comment by davevad93** [comment]
Hi @iCSawyer ,

I don't know why this happens, but you aren't the first one experiencing this: #131668 , #135388 , #135546 

Try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your problem. This user #136400 solved it by doing that.
